src/ctim/schemas/common.cljc

```diff
@@ -5,7 +5,7 @@
             [clojure.set :refer [map-invert]]
             [schema.core :as s]))
 
-(def ctim-schema-version "0.3.1")
+(def ctim-schema-version "0.4.0")
```

Comment:
Should be 0.3.3 to match the current CTIM version, I think (or we could bump the CTIM version).